apps-rendering/src/item.ts

```diff
@@ -323,24 +327,33 @@ const isPicture = hasTag('type/picture');
 
 const fromCapiLiveBlog =
 	(context: Context) =>
-	(request: RenderingRequest): LiveBlog | DeadBlog => {
+	(
+		request: RenderingRequest,
+		blockId: Option<string>,
+	): LiveBlog | DeadBlog => {
 		const { content } = request;
 		const body = content.blocks?.body ?? [];
+		const pageSize = content.tags.map((c) => c.id).includes('sport/sport')
+			? 30
+			: 10;
 
+		const parsedBlocks = parseLiveBlocks(body)(context);
+		const pagedBlocks = getPagedBlocks(pageSize, parsedBlocks, blockId);
 		return {
 			design:
 				content.fields?.liveBloggingNow === true
 					? ArticleDesign.LiveBlog
 					: ArticleDesign.DeadBlog,
-			blocks: parseLiveBlocks(body)(context),
+			blocks: parsedBlocks,
```

Comment:
I _think_, when we come to implementing the Pagination component in apps-rendering, we'll find that we don't need both `parsedBlocks` and `pagedBlocks`. Possibly we'll change to `blocks: pagedBlocks` because we only want to render the page of blocks, not all the blocks.